lib/wit.js

```diff
@@ -203,9 +203,9 @@ const validate = (opts) => {
   opts.witURL = opts.witURL || DEFAULT_WIT_URL;
   opts.apiVersion = opts.apiVersion || DEFAULT_API_VERSION;
   opts.headers = opts.headers || {
-    'Authorization': 'Bearer ' + opts.accessToken,
-    'Accept': 'application/vnd.wit.' + opts.apiVersion + '+json',
-    'Content-Type': 'application/json',
+    Authorization: 'Bearer ' + opts.accessToken,
```

Comment:
similarily, there is no difference between writing 'Authorization', vs Authorization: